Core/src/utils/Handler.cs

```diff
@@ -9,39 +9,49 @@
 
 namespace Core.utils
 {
-    public class FileHandler
+    public sealed class FileHandler
     {
 		/* singleton */
 		private FileHandler() { }
-		private static FileHandler singleton;
-		public static FileHandler getSingleton() {
-			if (singleton == null) singleton = new FileHandler();
-			return singleton;
+		private static readonly FileHandler singleton = new FileHandler();
+		public static FileHandler GetFileHandler {
```

Comment:
since the `GetFileHandler` is not a method remove the Get prefix
```suggestion
		public static FileHandler FileHandler {
```

Reply:
Member name cannot be the same as the type.

Reply:
Would FileHandlerInstance work?, I mean is that name good?

Reply:
Just Instance would be easy to use!

Reply:
**We should have to create our own coding conversion for BOOT-LOOP.**
and I'm on it.